examples/src/main/java/kafka/examples/ExactlyOnceMessageProcessor.java

```diff
@@ -170,6 +148,43 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
         latch.countDown();
     }
 
+    private int processMessages()
+        throws ProducerFencedException, FencedInstanceIdException {
```

Comment:
Seems like we don't allow a way to set the instanceId in this example, so does it make sense to include `FencedInstanceIdException`?

Reply:
It should be configured at consumer level, I could add it to the consumer config.